src/composables/useFlawAffectsModel.ts

```diff
@@ -39,13 +38,6 @@ export function useFlawAffectsModel() {
     initialAffects.value = deepCopyFromRaw(flaw.value.affects);
   }
 
-  function refreshAffects() {
-    return getFlaw(flaw.value.uuid).then((response) => {
-      flaw.value.affects = [...response.affects];
-      initialAffects.value = [...response.affects];
-    });
-  }
```

Comment:
We might need this still until we fully adopt patching Vue state with PUT responses, or with changes to trackers.

Reply:
Oh well you are right again, I need more sleep...

I think I tested this before the change in useFlaw to prevent updating the flaw, saw the duplicated query and removed it, which of course then does not work if the flaw is not updated duh

Reply:
I was doing some test in stage and this seems to do nothing...

The function is only called in two places:
https://github.com/RedHatProductSecurity/osim/blob/f064cfce07ed95a3b9df446a5eb003858e96e58a/src/composables/useFlawAffectsModel.ts#L263-L265
https://github.com/RedHatProductSecurity/osim/blob/f064cfce07ed95a3b9df446a5eb003858e96e58a/src/components/FlawAffects/FlawAffects.vue#L534-L539

The first one is redundant, since when we update/create an affect, it goes through `useFlawModel` which fetches the flaw/affects after, so this here only duplicates the requests.

The second one is not even used, as there's no `affects:refresh` event, it was removed in this refactor #377

Reply:
With these change, creating/deleting an affect works as expected. Since we are using the `useFlawModel` to fetch a new flaw/affects, we show the Loading icon for a moment, once we start using the response from creating the affect, this icon should not appear.

Creating trackers also works as expected, when a tracker is created, the `refreshRelatedFlaws` function is called, loading all the affects and thus updating both the trackers and affects table